https://issues.redhat.com/browse/ACM-19431 (#7728)
open-cluster-management/rhacm-docs: release_notes/acm_errata.adoc

* Finds the managed cluster with the label `local-cluster` set to `true` when checking for the subscription for the application clusters. (link:https://issues.redhat.com/browse/ACM-18538[ACM-18538])

> * Resolves a compatibility issue that prevented the  `MultiClusterObservability` custom resource definition (CRD) from being reapplied, and this issue caused the {acm-short} updates to get stuck. (link:https://issues.redhat.com/browse/ACM-19094[ACM-19094])

Reviewer: ```suggestion
* Resolves a compatibility issue that prevented the `MultiClusterObservability` custom resource definition (CRD) from being reapplied, which caused updates to get stuck. (link:https://issues.redhat.com/browse/ACM-19094[ACM-19094])
```
